src/chat/chatter.py

```diff
@@ -295,11 +296,13 @@ def pressed(self, item):
                 action_inspect_in_mordor = QtGui.QAction("Send the Orcs", menu)
                 menu.addAction(action_inspect_in_mordor)
 
+                ROOT = Settings.get('mordor/host')
+
                 def send_the_orcs():
                     if self.id != -1:
-                        QtGui.QDesktopServices.openUrl(QUrl("http://mordor.faforever.com/users/{}".format(self.id)))
+                        QtGui.QDesktopServices.openUrl(QUrl("http://" + ROOT + "/users/{}".format(self.id)))
```

Comment:
As a style note, `"{}".format(foo)` should be preferred over `"%s" % foo`, since it's less error-prone and the format syntax is more powerful (You can later rearrange presentation without rearranging order of arguments, it supports aligning when padding and more easily readable keyword arguments:

```python
data = {'first': 'Hodor', 'last': 'Hodor!'}
# Old
'%(first)s %(last)s' % data
# New
'{first} {last}'.format(**data)
```